modules/rubiconBidAdapter.js

```diff
@@ -228,6 +228,18 @@ export const spec = {
         'tk_user_key', userId
       ];
 
+      // add GDPR properties if enabled
+      if (config.getConfig('consentManagement')) {
+        if (bidRequest.gdprConsent && typeof bidRequest.gdprConsent === 'object') {
+          if (typeof bidRequest.gdprConsent.consentRequired === 'boolean') {
```

Comment:
@idettman In the upcoming changes to the consentManagement module, we'll be renaming this field to `bidderRequest.gdprConsent.gdprApplies`

Can you please make the necessary changes to point to the new field?   Please also ensure you're pointing to the `bidderRequest` object and not the `bidRequest` object.

As a final request, can you please make the above updates on the earlier set of GDPR code in the adapter file (for the video requests)?

Reply:
Sure thing, I'll get an update asap